### PR TITLE
Clarify account structure and user accounts

### PR DESCRIPTION
**Pull request by m-green**
### Context
We've had questions from service teams about how to set up their services in Pay.

### Changes proposed in this pull request
Rejig the content to help avoid confusion about team members having access to different services.

### Guidance to review
Please check if factually correct.

### CODE REVIEW

**Comment by m-green** [comment]
Hi @miriamalice - I've rethought this following our chat last week. I think there's still wider work we need to do on the page as a whole, but do you think this small change might help - where I've added (or technically moved) content about what need account type 1 meets? This means that every account type has a description of how it relates to user accounts.

File: source/account_structure/index.html.md.erb [hunk 1]
```diff
@@ -35,6 +28,12 @@ both GOV.UK Pay and the PSP.
 In this setup, payments to services can go to separate bank accounts because
 each service has a PSP account.
 
+You can:
+
+- give access for each service to different staff members
+- report separately on each of your services
+- optionally use different names on the GOV.UK Pay payment pages
```
Comment (miriamalice) [inline source/account_structure/index.html.md.erb]: Should we clarify - 'different service names'?

File: source/account_structure/index.html.md.erb [hunk 2]
```diff
@@ -35,6 +28,12 @@ both GOV.UK Pay and the PSP.
 In this setup, payments to services can go to separate bank accounts because
```
Comment (miriamalice) [inline source/account_structure/index.html.md.erb]: Would it be helpful to add this into the list of 'you can' things below?

File: source/account_structure/index.html.md.erb (hunk 1 above)
Comment (m-green) [reply source/account_structure/index.html.md.erb]: Good plan - added!

File: source/account_structure/index.html.md.erb (hunk 2 above)
Comment (m-green) [reply source/account_structure/index.html.md.erb]: Also a good plan - have added (and rejigged the bullet order so it better matches reality).

**Pull request merged**